### PR TITLE
Add current-for-tokens

### PR DESCRIPTION
**Pull request by pschorf**
## Changes proposed in this PR
- Add `current-for-tokens` to service details
- Add `run-using-marathon.sh`

## Why are we making these changes?
Allows users to determine if a service is the current version for a token.

### CODE REVIEW

File: waiter/src/waiter/handler.clj [hunk 1]
```diff
@@ -419,7 +430,9 @@
                      (:time service-suspended-state)
                      (assoc :service-suspended-state service-suspended-state)
                      (seq source-tokens-entries)
-                     (assoc :source-tokens source-tokens-entries))]
+                     (assoc :source-tokens source-tokens-entries)
+                     (seq current-for-tokens)
```
Comment (dposada) [inline waiter/src/waiter/handler.clj]: Don't we want to add this key even if `current-for-tokens` is empty?

File: waiter/src/waiter/handler.clj [hunk 2]
```diff
@@ -352,11 +352,21 @@
      :failed-instances (get service-id->failed-instances service-id)
      :killed-instances (get service-id->killed-instances service-id)}))
 
+(defn- get-current-for-tokens
+  [source-token-entries token->token-hash]
+  (->> source-token-entries
+       (reduce (fn [acc tokens] (into acc tokens)) [])
+       (filter (fn [{:strs [token version]}]
+                 (let [current-version (token->token-hash token)]
+                   (log/info "checking token" token ". Current version" current-version "source version" version)
```
Comment (dposada) [inline waiter/src/waiter/handler.clj]: `log/debug`?

File: waiter/integration/waiter/token_request_test.clj [hunk 3]
```diff
@@ -1358,6 +1358,36 @@
...
+
+          (testing "old service is no longer current for token"
+            (let [initial-service-details' (service-settings waiter-url initial-service-id)
+                  new-service-id (retrieve-service-id waiter-url {:x-waiter-token token})
+                  new-service-details (service-settings waiter-url new-service-id)]
+              (is (nil? (:current-for-tokens initial-service-details')))
+              (is (= [token] (:current-for-tokens new-service-details))))))
+        (finally
+          (delete-token-and-assert waiter-url token))))))
+
```
Comment (dposada) [inline waiter/integration/waiter/token_request_test.clj]: There's a good test for this in the CLI tests:

https://github.com/twosigma/waiter/blob/master/cli/integration/tests/waiter/test_cli.py#L1087-L1153

Should we replicate it here?

File: waiter/src/waiter/handler.clj (hunk 1 above)
Comment (pschorf) [reply waiter/src/waiter/handler.clj]: I was just following the convention of all the other optional stuff in the file. Even service description isn't added if it's empty or missing.

File: waiter/src/waiter/handler.clj (hunk 2 above)
Comment (pschorf) [reply waiter/src/waiter/handler.clj]: 👍 done

File: waiter/integration/waiter/token_request_test.clj (hunk 3 above)
Comment (pschorf) [reply waiter/integration/waiter/token_request_test.clj]: Done

**Pull request merged**